collections/_posts/2025/04/18/2025-04-18-meshery-extensions-org.md

```diff
@@ -0,0 +1,150 @@
+
+### Development Process
+
+- **Platform**: Follows a structured release cycle with [stable and edge channels](https://docs.meshery.io/project/contributing/build-and-release). Changes undergo rigorous review to maintain stability. Notify platform extenders and system integrators of upcoming changes in underlying framework to ensure time is afforded to maintain compatibility.
+- **Extensions**: Operate on independent release cycles, allowing rapid iteration. Developers use Meshery’s extension points to integrate with the core platform, following contribution guidelines (Meshery Contributing).
+
+### Integration Testing
+
+- **Compatibility Testing**: Extensions are tested against multiple core platform versions to ensure compatibility, using tools like MeshSync for resource discovery.
+- **Automated Pipelines**: GitHub Actions automate testing and snapshot generation, as seen in extensions like Helm Kanvas Snapshot (Meshery v0.8.0 Release).
+- **Performance Testing**: Meshery’s performance management features can be used to benchmark extensions, ensuring they meet efficiency standards.
+
+### Documentation and Onboarding
+
+- **Comprehensive Guides**: Documentation covers core platform usage, extension development, and integration (Meshery Docs). The Newcomers’ Guide and MeshMates program aid onboarding (Meshery Community).
+- **Catalog and Templates**: Meshery’s catalog of design templates includes extension configurations, promoting best practices (See Meshery Catalog).
+- **Community Resources**: Weekly meetings, Slack channels, and the community handbook provide ongoing support (See Community Handbook).
+
+## Reflections on Other Projects
+
+Meshery's expansion strategy mirrors successful models in other CNCF projects. BuildPacks and Argo https://github.com/argoproj-labs. Crossplane uses github.com/crossplane for its core platform and github.com/crossplane-contrib for community-contributed providers and functions. Similarly, Kubernetes maintains github.com/kubernetes for core components and github.com/kubernetes-sigs for Special Interest Groups (SIGs). These separations enable focused development, decentralized governance, and easier contribution, which Meshery aims to emulate.
+
+Autonomy: Extension teams have autonomy over their development processes, provided they adhere to Meshery’s code of conduct and integration guidelines, akin to Argo’s labs model where projects operate independently but align with broader project goals.
+
+### Crossplane
+
+Crossplane’s separation of github.com/crossplane and github.com/crossplane-contrib facilitates a clear distinction between the core platform and community contributions. Providers and functions in the contrib organization allow third-party developers to extend Crossplane’s capabilities without impacting core stability. This model supports Meshery’s approach by demonstrating how a separate organization can foster innovation while maintaining a reliable core.
```

Comment:
Crossplane’s separation of `github.com/crossplane` and `github.com/crossplane-contrib` facilitates a clear distinction between the core platform and community contributions. Providers and functions in the contrib organization allow third-party developers to extend Crossplane’s capabilities without impacting core stability. This model supports Meshery’s approach by demonstrating how a separate organization can foster innovation while maintaining a reliable core.